game.py

```diff
@@ -119,14 +119,17 @@ def load_remote(self, grid, data, mode, signal=False):
         self.model.data['running'] = 'False'
 
         for card in self.model.grid:
-            if card['state'] == '1':
-                self.emit('flip-card', self.model.grid.index(card), False)
-                self.last_flipped = self.model.grid.index(card)
-            elif card['state'] != '0':
-                stroke_color, fill_color = card['state'].split(',')
-                self.emit('flip-card', self.model.grid.index(card), False)
-                self.emit('set-border', self.model.grid.index(card),
-                          stroke_color, fill_color)
+            try:
```

Comment:
Introducing `try-catch` blocks will potentially hide issues and may cause unexpected behavior. 
Why is it needed ?

Reply:
@pro-panda
Try catch was introduced to identify empty card in the grid, which is the black tile. I shall replace the try catch blocks with a separate condition